### DISCUSSION TITLE
Audio

### DISCUSSION THREAD

**Discussion by Macsilencer** [tesla-android/issue-tracker#205] [General]
Hello

I hear nothing..

when connect the Android Bluetooth to my Tesla X 2017 Bluetooth...I have connection...eh, the Audio I don't have

i have download the OASP version, Android 13 on my Raspi4...and I have always and on all Apps System-Bluetooth Audio
what I can do then I can hear the System-Bluetooth Audio.....APP Audio switch to Bluetooth

many Thanks

**Comment by laugor94** [comment]
Hi I can heard the app on my Tesla from CarPlay 
I always have 2 sounds in parallel one from Tesla radio and one from CarPlay but its sound is not audible 
Could you give me an advice ? Or tip 
Rgds
Laurent
